inst.cc

```diff
@@ -15,3 +16,7 @@ template void
 internal::AffineConstraintsImplementation::set_zero_all(
   const std::vector<types::global_dof_index> &,
   LinearAlgebra::distributed::DynamicBlockVector<double> &);
+
+// Explicitly export instantiations to make polymorphic serialization work
+BOOST_CLASS_EXPORT(GrainTracker::RepresentationSpherical<2>)
+BOOST_CLASS_EXPORT(GrainTracker::RepresentationSpherical<3>)
```

Comment:
What is this?

Reply:
This is required for the polymorphic serialization to work, see https://www.boost.org/doc/libs/1_85_0/libs/serialization/doc/special.html. Actually, this is not properly working at the moment for tests, I need to fix it. One test crashes at the moment because of this issue. So this PR is not yet ready to merge.